fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -11868,8 +11868,8 @@
 | `value=(string 'false')`| (changed)| `"false"` |
 | `value=(string 'on')`| (changed)| `"on"` |
 | `value=(string 'off')`| (changed)| `"off"` |
-| `value=(symbol)`| (changed, error, warning, ssr error)| `` |
-| `value=(function)`| (changed, warning, ssr warning)| `"function f() {}"` |
+| `value=(symbol)`| (initial, warning, ssr error, ssr mismatch)| `<empty string>` |
+| `value=(function)`| (initial, warning, ssr mismatch)| `<empty string>` |
```

Comment:
Why do we have `ssr error` and `ssr mismatch` here but not in the other one? Can we fix SSR too?

Reply:
Yeah, I'm curious about that too. Writing a test for it now.